docs/sphinx/_toc.yml.in

```diff
@@ -0,0 +1,38 @@
+# Anywhere {branch} is used, the branch name will be substituted.
+# These comments will also be removed.
+defaults:
+  numbered: False
+  maxdepth: 7
+root: index
+subtrees:
+- entries:
+  - file: what-is-hipsparselt.md
+    title: What is hipSPARSELt?
+  - file: tutorials/quick-start/linux.md
+    title: Quick-start
+  - file: reference/index.md
+    title: API reference
+    subtrees:
+    - entries:
+      - file: reference/supported-functions.md
+        title: Supported functions
+      - file: reference/device-stream-manage.md
+        title: Device & stream management
+      - file: reference/storage-format.md
+        title: Storage formats
+      - file: doxygen/docBin/html/index
+        title: API library
```

Comment:
Try adding doxygen/docBin/html/group__library__module.html 
Let's see if all the modules can be listed in the TOC